docs/arista.eos.eos_acls_module.rst

```diff
@@ -3016,35 +3016,6 @@ Examples
 
     # Using deleted:
 
-    # Before state:
-    # -------------
-    # show running-config | section access-list
-    # ip access-list test1
-    #    10 permit ip 10.10.10.0/24 any ttl eq 200
-    #    20 permit ip 10.30.10.0/24 host 10.20.10.1
-    #    30 deny tcp host 10.10.20.1 eq finger www any syn log
-    #    40 permit ip any any
-    # ipv6 access-list test2
-    #     10 deny icmpv6 any any reject-route hop-limit eq 20
-
-    # !
-
-    - name: Delete provided configuration
```

Comment:
This is formatted incorrectly (the ACL to delete is not named), the correct one is below - looks like it was fixed in the next section but this incorrect one was left in.